docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -44,6 +44,31 @@ SCIM offers several advantages for managing users and teams in Grafana:
 - **Reduced administrative overhead**: Eliminate manual user management tasks and reduce the risk of human error
 - **Enhanced security**: Automatically disable access when users leave your organization
 
+## Configure SCIM in Grafana
+
+The table below describes all SCIM configuration options. Like any other Grafana configuration, you can apply these options as [environment variables](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#override-configuration-with-environment-variables).
+
+| Setting              | Required | Description                                                                                                                                                                                | Default |
+| -------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- |
+| `user_sync_enabled`  | Yes      | Enable SCIM user provisioning. When enabled, Grafana will create, update, and deactivate users based on SCIM requests from your identity provider.                                         | `true`  |
```

Comment:
Should we add that provisioned users cannot be deleted from Grafana?

Reply:
I think this probably should be part of the [User provisioning with SCIM](https://deploy-preview-grafana-102183-zb444pucvq-vp.a.run.app/docs/grafana/latest/setup-grafana/configure-security/configure-scim-provisioning/manage-users-teams/#user-provisioning-with-scim). Good point though, I will update.